utils/v2tofhirr4/utilities.bal

```diff
@@ -399,10 +399,22 @@ public isolated function nk1ToContact(hl7v2commons:Nk12 nk12, hl7v2commons:Nk14
     return [patientContact];
 }
 
+isolated function isADTMessage(hl7:Message message) returns boolean {
```

Comment:
Shall we have a utility method to something like isTransactionalMessage so that we can add the other messages as well if we have identified in future

Reply:
Fixed in https://github.com/ballerina-platform/module-ballerinax-health.hl7v2/commit/d497cbe6ad4e45a286b23ca00be31ad7c5fb32e1